build/charts/antrea/templates/_helpers.tpl

```diff
@@ -18,8 +18,40 @@
 {{- end }}
 {{- end -}}
 
-{{- define "antreaImage" -}}
+{{- define "antreaAgentImageTag" -}}
+{{- if .Values.agentImage.tag }}
+{{- .Values.agentImage.tag -}}
+{{- else if eq .Chart.AppVersion "latest" }}
+{{- print "latest" -}}
+{{- else }}
+{{- print "v" .Chart.AppVersion -}}
+{{- end }}
+{{- end -}}
+
+{{- define "antreaControllerImageTag" -}}
+{{- if .Values.controllerImage.tag }}
+{{- .Values.controllerImage.tag -}}
+{{- else if eq .Chart.AppVersion "latest" }}
+{{- print "latest" -}}
+{{- else }}
+{{- print "v" .Chart.AppVersion -}}
+{{- end }}
+{{- end -}}
+
+{{- define "antreaControllerImage" -}}
+{{- if .Values.image.repository }}
```

Comment:
I would prefer `{{- if .Values.image }}` if possible. I think it will evaluate to false if the object is empty?